bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -308,4 +480,75 @@ mod tests {
             }
         }
     }
+
+    unsafe extern "C" fn get_dummy_txctx(context: *mut ffi::evmc_context) -> ffi::evmc_tx_context {
+        ffi::evmc_tx_context {
+            tx_gas_price: ffi::evmc_uint256be { bytes: [0u8; 32] },
+            tx_origin: ffi::evmc_address { bytes: [0u8; 20] },
+            block_coinbase: ffi::evmc_address { bytes: [0u8; 20] },
+            block_number: 42,
+            block_timestamp: 235117,
+            block_gas_limit: 105023,
+            block_difficulty: ffi::evmc_uint256be { bytes: [0xaa; 32] },
+        }
+    }
+
+    // Update these when needed for tests
+    fn get_dummy_context() -> ffi::evmc_context {
+        ffi::evmc_context {
+            host: Box::into_raw(Box::new(ffi::evmc_host_interface {
+                account_exists: None,
+                get_storage: None,
+                set_storage: None,
+                get_balance: None,
+                get_code_size: None,
+                get_code_hash: None,
+                copy_code: None,
+                selfdestruct: None,
+                call: None,
+                get_tx_context: Some(get_dummy_txctx),
+                get_block_hash: None,
+                emit_log: None,
+            })),
+        }
+    }
+
+    fn get_dummy_message() -> ffi::evmc_message {
+        ffi::evmc_message {
+            kind: ffi::evmc_call_kind::EVMC_CALL,
+            flags: 0,
+            depth: 123,
+            gas: 105023,
+            destination: ffi::evmc_address { bytes: [0u8; 20] },
+            sender: ffi::evmc_address { bytes: [0u8; 20] },
+            input_data: std::ptr::null() as *const u8,
+            input_size: 0,
+            value: ffi::evmc_uint256be { bytes: [0u8; 32] },
+            create2_salt: ffi::evmc_uint256be { bytes: [0u8; 32] },
+        }
+    }
+
+    #[test]
+    fn execution_context() {
```

Comment:
Please test at last one function, for example `get_code_size`.

Reply:
Done in a separate test case.